solutions/swb-app/src/dataSetRoutes.ts

```diff
@@ -146,7 +147,10 @@ export function setUpDSRoutes(router: Router, dataSetService: DataSetPlugin): vo
     '/datasets',
     wrapAsync(async (req: Request, res: Response) => {
       const authenticatedUser = validateAndParse<AuthenticatedUser>(AuthenticatedUserParser, res.locals.user);
-      const response = await dataSetService.listDataSets(authenticatedUser);
+      const pageSize = toNumber(req.query.pageSize) || 100;
```

Comment:
nit: thought we were going with default 50. No issues with 100, just curious if there was a reason.

Reply:
I updated this but forgot to push i think